ui.content/src/main/content/jcr_root/conf/hclecomm/settings/wcm/templates/product-page-template/policies/.content.xml

```diff
@@ -14,7 +14,7 @@
                 jcr:primaryType="nt:unstructured"
                 sling:resourceType="wcm/core/components/policies/mapping"/>
             <responsivegrid
-                cq:policy="wcm/foundation/components/responsivegrid/policy_600116016309972"
+                cq:policy="wcm/foundation/components/responsivegrid/policy_6302423511085"
```

Comment:
@DeepaliAgrwl  wht is the change here?

Reply:
policy_6302423511085 is the actual hclecomm content policy that QA needs to select . Added the mapping to this policy.